app/Http/Controllers/WikiEntityImportController.php

```diff
@@ -34,7 +34,7 @@ public function create(Request $request): \Illuminate\Http\JsonResponse
             'entity_ids' => ['required', 'string', function (string $attr, mixed $value, \Closure $fail) {
                 $chunks = explode(',', $value);
                 foreach ($chunks as $chunk) {
-                    if (!preg_match("/^[A-Z]\d+$/", $chunk)) {
+                    if (!preg_match("/^[A-Z]\d+(@\d+)?$/", $chunk)) {
```

Comment:
Just double checking: is my understanding correct that revision ids will always be integers and nothing else?

Reply:
correct :)